AtomGPS_wigler.ino

```diff
@@ -57,8 +57,14 @@ void waitForGPSFix() {
 void initializeFile() {
   int fileNumber = 0;
   bool isNewFile = false;
+
+  // create a date stamp for the filename
+  char fileDateStamp[12];
```

Comment:
Increase the buffer size to 15 to be safe. 13 including the null terminator as is.

Reply:
Good point, will update